tasks/silenced.rb

```diff
@@ -0,0 +1,66 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+require 'json'
+require 'open3'
+require 'puppet'
+require 'tempfile'
+
+def sensuctl_create(name, spec, namespace)
```

Comment:
Does bolt have puppet string support? How is it meant to be documented?

Reply:
I believe the JSON metadata generates documentation on Forge. https://forge.puppet.com/puppetlabs/exec/tasks

The docs also mention tasks getting automatic docs from strings using JSON metadata: https://puppet.com/docs/puppet/5.5/puppet_strings_style.html#puppet-tasks-and-plans